client/components/popover/index.jsx

```diff
@@ -71,89 +71,101 @@ var Popover = React.createClass( {
 		return null;
 	},
 
-	_showOrHideTip: function( prevProps ) {
-		if ( ! this._tip ) {
-			this._tip = new Tip( '' );
-			this._tip.classname = this.props.className;
-			this._container = this._tip.inner;
+	_initTip: function() {
+		this._tip = new Tip( '' );
+		this._tip.classname = this.props.className;
+		this._container = this._tip.inner;
+	},
+
+	_renderTip: function() {
+		let content = <Content { ...omit( this.props, 'className' ) } onClickOutside={ this._onClickOutside } onClose={ this._close } />;
+
+		// Context is lost when creating a new render hierarchy, so ensure
+		// that we preserve the context that we care about
+		if ( this.context.store ) {
```

Comment:
I know this is how it used to work before, but do you think we should try to preserve any other context, apart from the redux one?